src/features/MoviePage/useMovieCredits.js

```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import { url_back, url_front_movie } from "../../common/API/requests";
+
+const url = `${url_front_movie}1096197/credits${url_back}`;
+
+export const useMovieCredits = () => {
+    const [movieCredits, setMovieCredits] = useState({
+        status: "loading",
+        data: [],
+    });
+
+    useEffect(() => {
+        const getMovieCredits = async () => {
+            try {
+                const response = await axios.get(url);
+                setMovieCredits({
+                    status: "success",
+                    data: response.data,
+                });
+            } catch (error) {
+                setMovieCredits({
+                    status: "error"
+                });
+                console.log(error);
+            }
+        };
+
+        setTimeout(getMovieCredits, 0);
```

Comment:
`setTimeout` is not needed if it has the value "0".

Reply:
I'm not sure if it is a bad practice, but deleting it causes issues, so I probably would've had to rewrite the function in a different way to solve it. Also I've read somewhere that setTimeout(0) schedules the callback after the shortest possible delay, so I think it works in this instance.